homeassistant/components/rainforest_eagle/sensor.py

```diff
@@ -16,59 +12,59 @@
     STATE_CLASS_MEASUREMENT,
     STATE_CLASS_TOTAL_INCREASING,
     SensorEntity,
+    SensorEntityDescription,
+    StateType,
 )
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
 from homeassistant.const import (
     CONF_IP_ADDRESS,
     DEVICE_CLASS_POWER,
     ENERGY_KILO_WATT_HOUR,
 )
-import homeassistant.helpers.config_validation as cv
-from homeassistant.util import Throttle
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import ConfigType
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
 
-CONF_CLOUD_ID = "cloud_id"
-CONF_INSTALL_CODE = "install_code"
-POWER_KILO_WATT = "kW"
+from .const import CONF_CLOUD_ID, CONF_INSTALL_CODE, DOMAIN
+from .data import EagleDataCoordinator
 
 _LOGGER = logging.getLogger(__name__)
 
-MIN_SCAN_INTERVAL = timedelta(seconds=30)
-
-
-@dataclass
-class SensorType:
-    """Rainforest sensor type."""
-
-    name: str
-    unit_of_measurement: str
-    device_class: str | None = None
-    state_class: str | None = None
-
+POWER_KILO_WATT = "kW"
```

Comment:
We have this common constant:
https://github.com/home-assistant/core/blob/e11ffbcdafca67b2d0f4d3059e41851dd9ea6c00/homeassistant/const.py#L413